src/Twig/EtudeExtension.php

```diff
@@ -228,7 +228,7 @@ public function getErrors(Etude $etude)
          *************************/
 
         // Description de l'AP suffisante
-        if (strlen($etude->getDescriptionPrestation()) < 300) {
+        if ($etude->getCeActive() != true and  strlen($etude->getDescriptionPrestation()) < 300 ) {
```

Comment:
`getCeActive()` retourne un booléen. Comparer avec true n'est donc pas nécessaire et la condition doit donc s'écrire `if(!$etude->getCeActive()`

Est-ce que tu peux utiliser `&&` et `||` (au lieu de `and` et `or`) pour les conjonctions et les disjonctions ? 

Raisons:
 * au niveau de PHP, de sombres histoires de précédences d'opérateurs.
 * au niveau de Jeyser comme tout a été passé en && et ||, de cohérence dans la base de code)